app/views/shared/_tax_return_assignees.html.erb

```diff
@@ -1,16 +1,25 @@
+<% status_updateable ||= false %>
```

Comment:
This isn't a pattern I see a lot in our codebase. I wonder if we could make the custom elements for updateable status listing be a `yield`, or make this parameter required. Non-blocking.

Reply:
I've been trying to treat partials as similar to components in react (wrapped in a class I can style with its own partial _scss file, having required/optional params, etc).